transferwiseclient/transferwiseclient.py

```diff
@@ -1,102 +1,136 @@
 import uuid
 import requests
 import json
+from dateutil.parser import parse
+from datetime import datetime
 
-def getTransferWiseProfiles(access_token):
-  # To get the personal profile ID, use json.loads(profiles.text)[0]['id']
 
-  profiles = requests.get('https://api.transferwise.com/v1/profiles',
-    headers={
-    'Authorization': 'Bearer '+ access_token,
-    'Content-Type': 'application/json'})
+class TransferWiseClient:
+  def __init__(self, access_token):
+    self.access_token = access_token
+    self.api_url = 'https://api.transferwise.com/v1/'
+    self.headers={'Authorization': 'Bearer '+ self.access_token,
+                  'Content-Type': 'application/json'}
+
+  def get(self, method, data = {}):
```

Comment:
Having a mutable default argument is usually frowned upon?

```python
def get(self, method, data=None):
    if data is None:
        data = {}
    ...
```
?

Reply:
A solution to mutable default arguments is to set them to None and check. Which is what I have done